core/src/components/popover/popover.tsx

```diff
@@ -260,7 +349,18 @@ export class Popover implements ComponentInterface, OverlayInterface {
     this.usersElement = await attachComponent(delegate, this.el, this.component, ['popover-viewport'], data, this.inline);
     await deepReady(this.usersElement);
 
-    this.currentTransition = present(this, 'popoverEnter', iosEnterAnimation, mdEnterAnimation, this.event);
+    this.configureKeyboardInteraction();
+    this.configureDismissInteraction();
+
+    this.currentTransition = present(this, 'popoverEnter', iosEnterAnimation, mdEnterAnimation, {
+      event: this.event || event,
+      size: this.size,
+      trigger: this.triggerEl,
+      reference: this.reference,
+      side: this.side,
+      align: this.alignment,
+      focusFirstDescendant: focusFirstItem
```

Comment:
Reasoning for changing names here? Might as well call the param "focusFirstDescendant" as well.